cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -1629,7 +1624,7 @@ func generateMountsModeRun(mst *initramfsMountsState) error {
 		return err
 	}
 
-	typs := []snap.Type{snap.TypeBase, snap.TypeKernel}
+	typs := []snap.Type{snap.TypeBase, snap.TypeGadget, snap.TypeKernel}
 
 	// 4.2 choose base and kernel snaps (this includes updating modeenv if
 	//     needed to try the base snap)
```

Comment:
this comment now should also mention the gadget?

Reply:
Fixed